base/deprecated.jl

```diff
@@ -1510,6 +1510,12 @@ end
 # Issue #26248
 @deprecate conj(x) x
 
+# PR #26436
+@deprecate equalto(x) isequal(x)
+@deprecate(occursin(x), in(x))
```

Comment:
Why the parens?

Reply:
Otherwise it parses as `occursin(x) in x`

Reply:
d'oh!